doc/usage_recomendation.rst

```diff
@@ -0,0 +1,46 @@
+.. _scenario: options.html#scenario
+.. _PCS: options.html#paramcs
+.. _TAE: tae.html
+
+Usage Recommendation
+===========
+*SMAC* of course itself offers a lot of design choices, some of which are crucial to achieve peak performance. Luckily, often it is enough to distinguish between a few problem classes.
+To make the usage of *SMAC* as easy as possible, we provide several facades designed for these different use cases. Here we give some general recommendations on
+when to use which facade. This recommendation is based on our experience and technical limitations and is by far not intended to be complete:
+
++-----------------------+----------+-----+-----+-----+-----+-----+-----+-----+-------+
+|                       | SMAC4BO  | SMAC4HPO              | SMAC4AC                 |
++=======================+==========+=====+=====+=====+=====+=====+=====+=====+=======+
+| # parameter           | low      | low/medium/high       | low/medium/high         |
++-----------------------+----------+-----+-----+-----+-----+-----+-----+-----+-------+
+| categorical parameter | no       | yes | yes | no  | no  | yes | yes | no  | no    |
++-----------------------+----------+-----+-----+-----+-----+-----+-----+-----+-------+
+| conditional parameter | no       | no  | yes | no  | yes | no  | yes | no  | yes   |
++-----------------------+----------+-----+-----+-----+-----+-----+-----+-----+-------+
+| instances             | no       | None or CV-folds      | yes                     |
++-----------------------+----------+-----+-----+-----+-----+-----+-----+-----+-------+
+| deterministic         | yes      | can handle both       | can handle both         |
```

Comment:
What is "both"?

Reply:
I thought of "both" for deterministic/non-deterministic, but now I changed it to
```| stochasticity         | no                   | supported             | supported               |```